app/users/users.go

```diff
@@ -32,21 +30,23 @@ type Retriever interface {
 
 // NewUserService returns UserService instance for retrieving or creating user records and accounts.
 func NewUserService() *UserService {
-	s := &UserService{}
-	s.updateLogger(monitor.F{})
+	s := &UserService{logger: monitor.NewModuleLogger("users")}
+	// s.updateLogger(monitor.F{})
 	return s
 }
 
-func (s *UserService) updateLogger(fields monitor.F) {
-	fields[monitor.TokenF] = s.token
-	s.log = monitor.NewModuleLogger("users").LogF(fields)
-}
+// func (s *UserService) updateLogger(fields monitor.F) {
+// 	fields[monitor.TokenF] = s.token
+// 	s.log = monitor.NewModuleLogger("users").LogF(fields)
+// }
 
 func (s *UserService) getLocalUser(id int) (*models.User, error) {
 	return models.Users(models.UserWhere.ID.EQ(id)).OneG()
 }
 
 func (s *UserService) createLocalUser(id int) (*models.User, error) {
+	log := s.logger.LogF(monitor.F{"id": id})
+
 	u := new(models.User)
```

Comment:
I know this is not part of the PR but how come you use new instead of the sqlboiler referenced way `u := &models.User{}`?

Reply:
Aren't those two constructs essentially identical?